public/scripts/menu.js

```diff
@@ -0,0 +1,7 @@
+export default async function initMenu(document, domUtils) {
+  document
+    .querySelector('#menuToggle svg')
+    .addEventListener('click', (e) =>
+      domUtils.toggleClass(document.querySelector('#menu'), 'hidden-768'),
```

Comment:
Why this strange class name `hidden-768`?

Reply:
it's hidden in 768px.
there is a hidden class already.

Reply:
Maybe would be better to name it as `hidden-gt-768px`?

Reply:
sure, if you think that's best.